assessment/asr_validation/wald2017b/analysis/A1-biaxial.i

```diff
@@ -15,432 +15,437 @@
 []
 
 [Variables]
-  [./T]
+  [T]
     order = FIRST
     family = LAGRANGE
     initial_condition = 10.6
-  [../]
-  [./rh]
+  []
+  [rh]
     order = FIRST
     family = LAGRANGE
     initial_condition = 0.8
     block = 1
-  [../]
+  []
+  [disp_x]
+  []
+  [disp_y]
+  []
+  [disp_z]
+  []
 []
 
 [AuxVariables]
-  [./resid_x]
-  [../]
-  [./resid_y]
-  [../]
-  [./resid_z]
-  [../]
-  [./ASR_ex]
+  [resid_x]
+  []
+  [resid_y]
+  []
+  [resid_z]
+  []
+  [ASR_ex]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./ASR_vstrain]
+  []
+  [ASR_vstrain]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./ASR_strain_xx]
+  []
+  [ASR_strain_xx]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./ASR_strain_yy]
+  []
+  [ASR_strain_yy]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./ASR_strain_zz]
+  []
+  [ASR_strain_zz]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./ASR_strain_xy]
+  []
+  [ASR_strain_xy]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./ASR_strain_yz]
+  []
+  [ASR_strain_yz]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./ASR_strain_zx]
+  []
+  [ASR_strain_zx]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./volumetric_strain]
+  []
+  [volumetric_strain]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./thermal_strain_xx]
+  []
+  [thermal_strain_xx]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./thermal_strain_yy]
+  []
+  [thermal_strain_yy]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./thermal_strain_zz]
+  []
+  [thermal_strain_zz]
     order = CONSTANT
     family = MONOMIAL
     block = 1
-  [../]
-  [./thermal_conductivity]
+  []
+  [thermal_conductivity]
     order = CONSTANT
     family = Monomial
-  [../]
-  [./thermal_capacity]
+  []
+  [thermal_capacity]
     order = CONSTANT
     family = Monomial
-  [../]
-  [./moisture_capacity]
+  []
+  [moisture_capacity]
     order = CONSTANT
     family = Monomial
-  [../]
-  [./humidity_diffusivity]
+  []
+  [humidity_diffusivity]
     order = CONSTANT
     family = Monomial
-  [../]
-  [./water_content]
+  []
+  [water_content]
     order = CONSTANT
     family = Monomial
-  [../]
-  [./water_hydrated]
+  []
+  [water_hydrated]
     order = CONSTANT
     family = Monomial
-  [../]
+  []
   [damage_index]
     order = CONSTANT
     family = MONOMIAL
   []
-  [./area]
+  [area]
     order = CONSTANT
     family = MONOMIAL
-  [../]
-  [./axial_stress]
+  []
+  [axial_stress]
     order = CONSTANT
     family = MONOMIAL
-  [../]
-  [./axial_strain]
+  []
+  [axial_strain]
     order = CONSTANT
     family = MONOMIAL
-  [../]
+  []
 []
 
 [Modules/TensorMechanics/Master]
-  [./concrete]
+  [concrete]
     block = 1
     strain = FINITE
-    add_variables = true
```

Comment:
So are we getting rid of `add_variables`?

Reply:
No. Just in inputs that use the line element action, which does not properly coordinate with the TMA. Going forward we might want to improve that action.

Reply:
Oh, I see...

Reply:
So we have to remove it from both the TM master and the line element action if we're using them together.

Reply:
yeah